javapeople.yaml

```diff
@@ -687,6 +687,12 @@
   twitter: "lbkulinski"
   fediverse: "lbkulinski@mastodon.world"
   github: "lbkulinski"
+- name: "Loïc Mathieu"
+  twitter: "loicmathieu"
+  fediverse: "loicmathieu@mastodon.online"
+  bluesky: "loicmathieu.fr"
+  github: "loicmathieu"
+  linkedin: "loïc-mathieu-475b144"
```

Comment:
The LinkedIn handle pattern appears to be not flexible enough in this case 🙂 

https://github.com/marchof/javabubble/blob/e3dc84e0892e1f95a706f742445f8d388cc15a0a/generator/src/main/java/org/javabubble/generator/model/LinkedInHandle.java#L7

According to [the docs](https://www.linkedin.com/help/linkedin/answer/a542685/manage-your-public-profile-url):

> Your custom URL can be 3-100 characters long and should not contain spaces, symbols, special characters, or the word LinkedIn.

We might align the implementation to such rules.

Reply:
If it's an issue, you can remove my LinkedIn account

Reply:
No, we fix it!

Reply:
I switched to the word class with Unicode support, which is good enough for umlaut characters.